lib/method_object.rb

```diff
@@ -1,53 +1,165 @@
+# frozen_string_literal: true
```

Comment:
`frozen_string_literal` may not work in older versions of Ruby like 2.1? Should check.